ta/sims/include/ta_sims.h

```diff
@@ -37,5 +37,7 @@ TEE_Result sims_read(uint32_t nParamTypes, TEE_Param pParams[4]);
 TEE_Result sims_write(uint32_t nParamTypes, TEE_Param pParams[4]);
 TEE_Result sims_get_counter(void *sesion_context, uint32_t param_types,
 			    TEE_Param params[4]);
+TEE_Result sims_entry_panic(void *sesion_context, uint32_t nParamTypes,
```

Comment:
typo: s/sesion_context/session_context/
Note you can fix the one above also (in the same commit I mean).

Reply:
Done